src/scripts/main.js

```diff
@@ -1,3 +1,17 @@
 'use strict';
 
-// write your code here
+const population = document.querySelectorAll('.population');
+
+const populationArray = [...population].map(element =>
+  +(element.innerText.split(',').join('')));
+
+const allPopulation = populationArray.reduce((sum, element) =>
+  (sum + element), 0);
+
+const averagePopulation = allPopulation / populationArray.length;
+
+const totalElement = document.querySelector('.total-population');
+const averageElement = document.querySelector('.average-population');
+
+totalElement.innerText = allPopulation.toLocaleString();
+averageElement.innerText = averagePopulation.toLocaleString();
```

Comment:
You should return the number in the format 452,543,964 instead of 452 543 964.
Try to change locale to 'en-US'